shell/platform/windows/accessibility_bridge_windows.cc

```diff
@@ -41,10 +41,13 @@ void AccessibilityBridgeWindows::OnAccessibilityEvent(
       DispatchWinAccessibilityEvent(win_delegate,
                                     ax::mojom::Event::kChildrenChanged);
       break;
-    case ui::AXEventGenerator::Event::DOCUMENT_SELECTION_CHANGED:
-      DispatchWinAccessibilityEvent(
-          win_delegate, ax::mojom::Event::kDocumentSelectionChanged);
-      break;
+    case ui::AXEventGenerator::Event::DOCUMENT_SELECTION_CHANGED: {
+        ui::AXNode::AXID focus_id = GetAXTreeData().sel_focus_object_id;
+        auto focus_delegate = GetFlutterPlatformNodeDelegateFromID(focus_id).lock();
+        DispatchWinAccessibilityEvent(
+            std::static_pointer_cast<FlutterPlatformNodeDelegateWindows>(focus_delegate), ax::mojom::Event::kDocumentSelectionChanged);
+        break;
+      }
```

Comment:
Document selection change event should be fired on the node containing the changed selection, not the root.